README.rst

```diff
@@ -1,19 +1,25 @@
 .. image:: https://translate.fedoraproject.org/widgets/dnf/-/dnf-master/svg-badge.svg
     :alt: Translation status
     :target: https://translate.fedoraproject.org/engage/dnf/?utm_source=widget
-###############
- Dandified YUM
-###############
+
+==================================================================================================================================================================================================
+⚠️ **Notice:** This repository is for **DNF4** only For the current DNF version (DNF5), visit `https://github.com/rpm-software-management/dnf5`
```

Comment:
I think there is a period missing between "only For"

Reply:
Also the link looks a bit odd here, maybe shorten the link text to `rpm-software-management/dnf5` or rephrase it compeletely, e.g. "The current DNF version is `DNF 5`." or "The current DNF version 5 has its `own repository`" (With the `ticked` words as the same link of course.)

But these are just suggestions to make it look nicer, your current PR would also be good enough for me.

Reply:
english is not my first language can you rewrite that bit for me ?

Reply:
> english is not my first language can you rewrite that bit for me ?

ohh i see the mistake now, no need to write, i am doing it myself